internal/apif/config.go

```diff
@@ -0,0 +1,57 @@
+package apif
+
+import "github.com/saucelabs/saucectl/internal/config"
+
+// Config descriptors.
+var (
+	// Kind represents the type definition of this config.
+	Kind = "apif"
```

Comment:
Just leaving it as a food for thought:

@KunalJain How is the product referred to now vs in the future? I don't see any APIF branding anymore.
I've seen two places on the UI that mentioned "APIF TEST" (that's the file import when creating a test) and Apifctl (the CLI). With their CLI replaced, that just leaves the "APIF TEST" button and perhaps some other places I missed.

TLDR; does our reference to our API Testing area as APIF still make sense? Or should we choose another moniker?

Reply:
The product will be referenced as API Testing going forward. We are moving away from the APIF brand.

Reply:
@mhan83 Maybe we should refer to it as `apitest` then? Other suggestions are welcome :D

Reply:
Let's go with `apitest`. `apit` just doesn't seem right for some reason...